{{cookiecutter.project_dashed}}-src/{{cookiecutter.project_namespace}}/config.py

```diff
@@ -34,9 +43,14 @@ class TestProfile(object):
     KEG_KEYRING_ENABLE = False
 
     # Mail related tests need to have this set, even though actual email is not generated.
-    MAIL_DEFAULT_SENDER = '{{cookiecutter.developer_email}}'
+    MAIL_DEFAULT_SENDER = 'test@test.com'
 
     CELERY = {
         # This should be for the docker container setup in the CircleCI config.
         'broker_url': 'amqp://guest@localhost:5672//',
     }
+
+    KEG_DB_ENGINE_OPTIONS = {
```

Comment:
Commented out above but not here.

Reply:
This is requires for the CI tests. My understanding is that we don't want to create a config file to run the tests in CI and this `TestProfile` is how we make those settings. LMK if you want something else.

Reply:
My question upon reading that comment would be: Why is this required in CI but not in development?  Why would CI require a config setting that a dev environment doesn't?  It would confuse me.

Reply:
Because the JSON changes are required for CI since we have tests for them. But they aren't on by default in our apps. I can flip it so that they are on by default. It doesn't matter to me one way or another.